project/profiles/capacity4more/behat/features/contact.feature

```diff
@@ -16,3 +16,21 @@ Feature: Contact
     And I fill in "E-mail addresses" with "info@example.com, other@example.com"
     And I press "Save configuration"
     Then I should see "The configuration options have been saved."
+
+  @api
+  Scenario: Check contact page as anonymous
+    Given I am an anonymous user
+    When I go to "contact"
+    And I press "Send message"
+    Then I should see "Subject field is required"
+    And I should see "Message field is required."
+    And I should see "Your name field is required."
+    And I should see "Your e-mail address field is required."
+
+  @api
+  Scenario: Check contact page as logged in user
+    Given I am logged in as user "isaacnewton"
+    When I go to "contact"
+    And I press "Send message"
+    Then I should see "Subject field is required"
+    And I should see "Message field is required."
```

Comment:
@DavidHernandez 
please add the other two lines as well in here but the other way:
```gherkin
And I should not see "Your name field is required."
And I should not see "Your e-mail address field is required."
```